### ISSUE TITLE
Urlscan-py always asking for API key even when value stored in DB

### ISSUE THREAD

**Issue by heywoodlh** [heywoodlh/urlscan-py#9]
Happening on MacOS at least. Not sure if it's across the board.

**Comment by heywoodlh** [comment]
Can confirm it's happening on Linux:

```
/ # urlscan init --api xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
/ # urlscan scan --url https://the-empire.systems
Please enter API key: 
```

**Comment by heywoodlh** [comment]
This has been fixed with most recent commit: d600ac5e64da557efca178e3cd48ccb0f8727100
